source/whats-new.txt

```diff
@@ -0,0 +1,54 @@
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+* :ref:`Version 4.2 <version-4.2>`
+* :ref:`Version 4.1 <version-4.1>`
+* :ref:`Version 4.0 <version-4.0>`
+
+
+.. _version-4.2:
+
+What's New in 4.2
+-----------------
+New features of the 4.2 Java driver release include:
+
+- Enabled Azure and GCP key stores to client-side field level encryption
+- Enabled Kerberos caching tickets for reuse in multiple authentication requests
+- `com.mongodb.MongoClient <http://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/MongoClients.html>`_ instances with ``MongoClientSettings`` or ``ConnectionString`` as the configuration
+-  ``explain()`` can now be used on `find <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`_ and `aggregate <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/AggregateIterable.html#explain()>`_ commands
+- Added `JsonObject <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/bson/org/bson/json/JsonObject.html#%3Cinit%3E(java.lang.String)>`_ class to make encoding from and decoding to JSON more efficient by avoiding an intermediate Map representation
+- Added `BsonRepresentation <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/bson/org/bson/codecs/pojo/annotations/BsonRepresentation.html>`_ annotation that allows ``ObjectId`` BSON values to be represented as a ``String`` in `POJO <https://en.wikipedia.org/wiki/Plain_old_Java_object>`_ classes
+- Added `Filters.empty() <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-core/com/mongodb/client/model/Filters.html#empty()>`_ method
+
+.. important::
+
+   There are new changes that may affect your application. Refer to the `Upgrading Guide <https://mongodb.github.io/mongo-java-driver/4.2/upgrading/>`_.
+
+.. _version-4.1:
+
+What's New in 4.1
+-----------------
+New features of the 4.1 Java driver release include:
+
+- Significant reduction in client-perceived failover times during planned maintenance events
+- `update() <https://mongodb.github.io/mongo-java-driver/4.1/apidocs/mongodb-driver-core/com/mongodb/client/model/UpdateOptions.html#hint(org.bson.conversions.Bson)>`_ and `delete() <https://mongodb.github.io/mongo-java-driver/4.1/apidocs/mongodb-driver-core/com/mongodb/client/model/DeleteOptions.html#hint(org.bson.conversions.Bson)>`_ support index hinting
+- ``find()`` supports `allowDiskUse() <https://mongodb.github.io/mongo-java-driver/4.1/apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#allowDiskUse(java.lang.Boolean)>`_ for sorts that require too much memory to execute in RAM
+- Authentication supports the MONGODB-AWS mechanism using Amazon Web Services (AWS) Identity and Access Management (IAM) credentials
+- Authentication requires fewer round trips to the server, resulting in faster connection setup
+
```

Comment:
One final thought: if you call out breaking changes in an .. important:: box, does it make sense to have a .. note:: here that says "this release has no breaking changes"? Perhaps that's just silly, as it's assumed?

Reply:
hm, there are notes about each update in that link actually, so it applies to all of them